front_end/html/edit_exercise.html

```diff
@@ -69,7 +134,25 @@ <h2>Create exercise</h2>
                 answer_code_editor.getSession().setMode(json_data["code_completion_path"]);
                 starter_code_editor.getSession().setMode(json_data["code_completion_path"]);
                 test_code_editor.getSession().setMode(json_data["code_completion_path"]);
+                check_code_editor.getSession().setMode(json_data["code_completion_path"]);
             });
+
+            // load back_end specific instructions for solution and test_code
+            let back_end_type = document.getElementById("back_end").value;
+            if (exampleDict[back_end_type + "_solution_instructions"] != "")
+              // split and capitalize back end, if applicable
+              document.getElementById("solution_additional_instructions").innerHTML = `<hr><p><strong>
+                ${(back_end_type.includes("_")) ? back_end_type.split("_").join(" ").replace(/\w\S*/g, (w) => (w.replace(/^\w/, (c) => c.toUpperCase()))) : back_end_type.replace(/\w\S*/g, (w) => (w.replace(/^\w/, (c) => c.toUpperCase())))}
+                Specific Instructions:</strong></p>` + exampleDict[back_end_type + "_solution_instructions"]
+            else
+              document.getElementById("solution_additional_instructions").innerHTML = ""
+            if (exampleDict[back_end_type + "_test_code_instructions"] != "")
+              // split and capitalize back end, if applicable
+              document.getElementById("test_code_additional_instructions").innerHTML = `<hr><p><strong>
+                ${(back_end_type.includes("_")) ? back_end_type.split("_").join(" ").replace(/\w\S*/g, (w) => (w.replace(/^\w/, (c) => c.toUpperCase()))) : back_end_type.replace(/\w\S*/g, (w) => (w.replace(/^\w/, (c) => c.toUpperCase())))}
+                Specific Instructions:</strong></p>` + exampleDict[back_end_type + "_test_code_instructions"]
+            else
+              document.getElementById("test_code_additional_instructions").innerHTML = ""
```

Comment:
<img width="1280" alt="image" src="https://user-images.githubusercontent.com/61362908/120704031-4e5e0600-c473-11eb-9d61-1d20dc2bea4f.png">
here is a screenshot of the UI, for reference

Reply:
<img width="1280" alt="image" src="https://user-images.githubusercontent.com/61362908/120908475-f0beea80-c627-11eb-9461-7adec160e9d5.png">
<img width="1280" alt="image" src="https://user-images.githubusercontent.com/61362908/120908478-f9172580-c627-11eb-8de4-f1231ce92b20.png">

Here is an updated view of the UI with back end specific instructions and toggled example code